lib/models/organisation.js

```diff
@@ -217,98 +217,108 @@ schema.post('init', function handleInit() {
 schema.pre('save', async function preSave(next) {
   const User = getConnection().model('User');
   const Role = getConnection().model('Role');
+  const organisation = this;
+  const objectId = mongoose.Types.ObjectId;
 
   // https://github.com/Automattic/mongoose/issues/1474
   this.wasNew = this.isNew;
```

Comment:
In our case, we have no postSave hook. Why are we using this trick?

Reply:
Yeah if we have no post save then it appears this is no longer necessary, it's possible we did have a post save hook in the past. Happy for you to remove this line if you can't find references to this `wasNew` property anywhere else in the codebase.

Reply:
Please comment on this thread with what you decide (if you decide to remove or keep this line) then resolve this comment.

Reply:
Actually I can see this being used a bit further down, but it's used within this function so we could just use `this.isNew` in its place.

Reply:
I think we need to delete this line of code for better readability. If we need to add a postSave hook in the future, just return this line of code.